### PR TITLE
Starting on writing some guidelines for checkout discussion sessions

### PR DESCRIPTION
**Pull request by karinlag**
This addresses issue #72 .

Can especially @gvwilson and @rgaiacs have a look? All comments and thoughts welcome.

### CODE REVIEW

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* Give an orientation of how the checkout procedure itself goes
+	* A signup etherpad will be sent out. Choose your subject, and sign up for a 
+session. Around 5 people gather on a hangout with one of the trainers. Each person will have 5 minutes to present a section of the material. The trainer will dictate where in the material the student should start. After presentation, there will be 5 minutes for
+	feedback from the trainer. All will be notified afterwards by email whether they
+	passed or not.
+	Tell people that it is often useful to have the material available on paper.
+* Make sure that all know where the lesson material can be found. Ensure that they all know about the instructor guide. 
+* Inform about known problems with the lesson (i.e. tell people that the python lesson is too long and that it is in the process of being cut). 
+* Go through each of the modules in the lesson. Ask if anybody have any problems, questions or comments about the module. Do anybody have thoughts about changes or additions or things that can be removed?
```
Comment (wking) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: “Do anybody” → “Does anybody”.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#Guidelines for checkout discussion sessions
+
+This document outlines some things that can be discussed during the checkout discussion sessions. The sessions are meant to be student driven, but some groups may be quieter than others. This is by no means an exhaustive list, other subjects are bound to come up. 
+
+## Beforehand:
+* Discussion lead creates etherpad. Post link to etherpad in the session list etherpad
```
Comment (rgaiacs) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: I thought that the etherpad will be create by the instructor trainer.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* Discussion lead creates hangout. Post link to hangout in the session list etherpad.
```
Comment (rgaiacs) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: I thought that we would use BlueJeans for this.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## During:
+* Inform students that a summary of the discussion will be kept in the etherpad, and that ideas and issues may be culled for future use by the SCF.
```
Comment (rgaiacs) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: Maybe reword to

> Inform students that a few notes could be take in the etherpad during the discussion, and that ideas and issues may be culled for future use by the SCF.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+* Give an orientation of how the checkout procedure itself goes
+	* A signup etherpad will be sent out. Choose your subject, and sign up for a 
+session. Around 5 people gather on a hangout with one of the trainers. Each person will have 5 minutes to present a section of the material. The trainer will dictate where in the material the student should start. After presentation, there will be 5 minutes for
+	feedback from the trainer. All will be notified afterwards by email whether they
+	passed or not.
+	Tell people that it is often useful to have the material available on paper.
+* Make sure that all know where the lesson material can be found. Ensure that they all know about the instructor guide. 
+* Inform about known problems with the lesson (i.e. tell people that the python lesson is too long and that it is in the process of being cut).
```
Comment (rgaiacs) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: "(i.e. tell people that the python lesson is too long and that it is in the process of being cut)" -> "(e.g. tell people that the Python/Shell lesson is too long for only one morning/afternoon)"

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 1 above)
Comment (rgaiacs) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: Can we suggest to do this only if attends don't have questions?

Comment (karinlag) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: I still suggest that we do this, even if just briefly, in case we have somebody on the quieter side who have a question to the material but who are not prone to interrupt.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 3 above)
Comment (karinlag) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: I personally prefer hangouts. If we are to use BlueJeans, that means that somebody in SCF needs to do things beforehand. Hangouts can be created by each discussion session leader on their own.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 5 above)
Comment (karinlag) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: +1

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 3 above)
Comment (rgaiacs) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: @gvwilson or @jduckles Could you clear this?

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 2 above)
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: I was planning to have everyone use the sign-up Etherpad so that comments from all groups would be visible to all others.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 3 above)
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: There are no time clashes between meetings, so the persistent BlueJeans meeting https://bluejeans.com/430106525
 can be used.  If discussion leaders would rather use Hangouts, that's OK too.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 4 above)
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: "Remind students that the Etherpad is a public, persistent document."

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+This document outlines some things that can be discussed during the checkout discussion sessions. The sessions are meant to be student driven, but some groups may be quieter than others. This is by no means an exhaustive list, other subjects are bound to come up. 
+
+## Beforehand:
+* Discussion lead creates etherpad. Post link to etherpad in the session list etherpad
+* Discussion lead creates hangout. Post link to hangout in the session list etherpad.
+
+## During:
+* Inform students that a summary of the discussion will be kept in the etherpad, and that ideas and issues may be culled for future use by the SCF.
+* Give an orientation of how the checkout procedure itself goes
+	* A signup etherpad will be sent out. Choose your subject, and sign up for a
```
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: Unneeded - they'll already be in the discussion.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+## Beforehand:
+* Discussion lead creates etherpad. Post link to etherpad in the session list etherpad
+* Discussion lead creates hangout. Post link to hangout in the session list etherpad.
+
+## During:
+* Inform students that a summary of the discussion will be kept in the etherpad, and that ideas and issues may be culled for future use by the SCF.
+* Give an orientation of how the checkout procedure itself goes
+	* A signup etherpad will be sent out. Choose your subject, and sign up for a 
+session. Around 5 people gather on a hangout with one of the trainers. Each person will have 5 minutes to present a section of the material. The trainer will dictate where in the material the student should start. After presentation, there will be 5 minutes for
+	feedback from the trainer. All will be notified afterwards by email whether they
```
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: why the indentation?

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+	passed or not.
+	Tell people that it is often useful to have the material available on paper.
```
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: :-1: - instead, inform people that they are expected to teach the 5 minutes *with only occasional references to notes (if any)*.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 5 above)
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: :-1: This is the sort of thing we want *them* to bring to *us* - discussion leaders should ask, "How long do you think the material actually takes to teach and why?" or wait for unprompted questions from participants of the form, "Isn't this a little... long?"

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 2 above)
Comment (karinlag) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: Sounds good.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 5 above)
Comment (karinlag) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: ....how about instead telling them that we know the material is not perfect, that it is under continuous development, and then tell them that suggestions are welcome?

My main point behind this to make them aware that the material is not perfect, and that they should employ their minds when teaching from them. This could for instance help novice python teachers become aware of the fact that they can´t aim to teach the material as is if they only have a morning for it, they have to cull it.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 7 above)
Comment (karinlag) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: The indentation is to highlight what the discussion lead should say when telling people how the checkout procedure takes place.

File: subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md (hunk 5 above)
Comment (gvwilson) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: Telling them that the material is not perfect is fine, but I'd like them to figure out for themselves that it's too long, and indeed what the other flaws are or might be.  (I also think that instead of saying "the material is not perfect", which sounds like we're defensive, we could emphasize that we're constantly looking for ways to improve the material.)

Comment (karinlag) [inline subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md]: I´ll try to incorporate that in a positive way :)

**Comment by karinlag** [comment]
I believe my latest update cleared up quite a few issues, and I hope it´s ready to go.
